### ISSUE TITLE
comparandum vs. value in file picking rules

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2005]
Creating a file picking rule and was thrown off by the use of "comparandum" in the file picking rules UI, when I was expecting to see "value" (especially because "value" is the field label that shows up in the list of file picking rules in Kiosk and because it is what shows up in the file picking explanation in the wiki). Not sure if it used to be "value" before? Is there a reason why "comparandum" is used now?

<img width="1504" alt="Screenshot 2023-02-28 at 7 28 58 AM" src="https://user-images.githubusercontent.com/67970402/221868545-0a8485f7-11bc-4b40-976b-dc397be29d05.png">

**Comment by urapadmin** [comment]
yes, it used to be "value" before but I must have made some of these barren UIs here a bit easier to understand and use (I really cannot remember when I did that). Hence different labels and some help texts and selections. "Value" seemed a bit unspecific. So I went for the fancy "comparandum". Don't like it? Or is it only about incoherent usage? In that case I'd apply it to the list view as well.

**Comment by luizaogs** [comment]
I think value seems clearer to me but that may very well just be because I was used to it, so comparandum should be fine as long as it’s consistent.
